python/src/scipp/table.py

```diff
@@ -176,6 +176,20 @@ def _table_from_dict_of_variables(dict_of_variables,
     return html
 
 
+def _is_bin_centers(container, var, dim):
+    """
+    Is var considered to be bin_centers in dimension dim
+    Considers all meta including non-dimensional meta
+
+    :param container: scipp object defining all meta
+    :param var: variable to determine if bin centers
+    :param dim: dimension to consider
+    :return:
```

Comment:
Not sure you need to fully specify the parameters in the docstring for a private function that begins with a `_`.
But if you want to keep it, you should specify the return value/type.

Reply:
Indeed, it's never going to get used directly - though I thought it would be some help to future maintainers to describe what was going on. I'll add the return info for completeness.